frontend/device/android/device.lua

```diff
@@ -362,13 +362,41 @@ function Device:canExecuteScript(file)
     end
 end
 
+--swallow all events
+local function processEvents()
```

Comment:
Probably not an objection, but this feels like something that might be cleaner as part of luajit-launcher? I'll leave it to @pazos.

Reply:
Looks like a duplicate of https://github.com/koreader/koreader-base/blob/master/ffi/input_android.lua#L139-L165 but without generating linux input events.

I'm sure there're different ways of doing this but not sure which is the best one.

Reply:
Oh right, I forgot about that one. That also feels cleaner to me. The effect is the same etc. but it feels more logical to me.

Reply:
@Frenzie: I would say LGTM as is. In the future we can refactor as a base/input function.

I don't want to do it at the moment :(